### PR TITLE
upgraded

### PR DESCRIPTION
**Pull request by himanshu-uniyal**
# Description

Please explain the changes you made here and link to any relevant issues.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -25,9 +25,15 @@ module "labels" {
 # Module      : ECR  REPOSITORY
 # Description : Provides an Elastic Container Registry Repository.
 resource "aws_ecr_repository" "default" {
-  count = var.enabled_ecr ? 1 : 0
-  name  = module.labels.id
-  tags  = module.labels.tags
+  count                = var.enabled_ecr ? 1 : 0
+  name                 = module.labels.id
+  tags                 = module.labels.tags
+  image_tag_mutability = "MUTABLE"
```
Comment (anmolnagpal) [inline main.tf:31]: Why its hard coded, make it variable

File: main.tf [hunk 2]
```diff
@@ -25,9 +25,15 @@ module "labels" {
...
+
+  image_scanning_configuration {
+    scan_on_push = true
```
Comment (anmolnagpal) [inline main.tf:34]: make this variable

**Review by anmolnagpal — CHANGES_REQUESTED** [review]

**Comment by anmolnagpal** [comment]
Enable encryption_configuration

File: main.tf (hunk 1 above)
Comment (himanshu-uniyal) [reply main.tf]: done sir

File: main.tf (hunk 2 above)
Comment (himanshu-uniyal) [reply main.tf]: done sir

**Pull request closed without merging**